seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/sink/JdbcExactlyOnceSinkWriter.java

```diff
@@ -171,7 +172,7 @@ private void beginTx() throws IOException {
         try {
             xaFacade.start(currentXid);
         } catch (Exception e) {
-            ExceptionUtils.rethrowIOException(e);
+            throw new JdbcConnectorException(JdbcConnectorErrorCode.XA_EXCEPTION, e);
```

Comment:
add Info "unable to start xa transaction"

Reply:
ok